nectar/downloaders/threaded.py

```diff
@@ -160,6 +160,7 @@ def requests_kwargs_from_nectar_config(config):
                                          host=parsed_url.host, port=config.proxy_port)
             requests_kwargs['proxies'] = {'http': parsed_url.url, 'https': parsed_url.url}
 
+        requests_kwargs['stream'] = True # required for reading the download in chunks
```

Comment:
@alexjfisher thank you for finding this issue and resolving it.  Sorry for the delay getting this reviewed but we all get jammed up some weeks.  I wanted to dig into a bit and make sure I really understood how this was broken and what the options were for fixing.  At first, I suspected a bug in `requests` but turned out to be simpler.